tests/test_config.py

```diff
@@ -1918,6 +1918,19 @@ def _test_quick_search_with_rcs(self,
         with self.assertRaises(TritonModelAnalyzerException):
             self._evaluate_config(new_args, yaml_content, subcommand='profile')
 
+    def test_bls_submodels(self):
+        """
+        Test that BLS submodels can be specified only per model in the YAML
```

Comment:
This comment doesn't seem to line up with what is happening.

Reply:
Fixed